crates/forge/bin/cmd/install.rs

```diff
@@ -505,6 +505,7 @@ mod tests {
     use tempfile::tempdir;
 
     #[test]
+    #[ignore = "slow"]
```

Comment:
I don't really want these tests in unit tests, same for some that call providers or fork.